airflow/utils/setup_teardown.py

```diff
@@ -181,7 +181,10 @@ def _push_tasks(cls, operator: AbstractOperator | list[AbstractOperator], setup:
 
     @classmethod
     def _update_teardown_downstream(cls, operator: AbstractOperator | list[AbstractOperator]):
-        """This recursively goes through the tasks downstream of the setup in the context manager,
+        """
+        Update _teardown_downstream_of_setup.
```

Comment:
@dstandish Any idea to better describe this function?

Reply:
I think I've found suitable phrasing here and for the other method below.